lfucg-exactions/client/js/components/LotForm.js

```diff
@@ -523,6 +523,7 @@ function mapDispatchToProps(dispatch, params) {
             event.preventDefault();
             if (selectedLot) {
                 dispatch(putLot(selectedLot));
+                hashHistory.push(`lot/form/${selectedLot}`);
```

Comment:
This will not work. It does not allow you to push to the page you are already on.